src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Rate/Rate.ts

```diff
@@ -318,6 +321,7 @@ export var directiveFactory = (template : string, adapter : IRateAdapter<RIRateV
                             })
                             .then(() => undefined);
                     }).finally<void>(() => {
+                        scope.isCast = true;
```

Comment:
This way the results will only show up when I click. When I reload the page I will still have rated, but I cannot see the results anymore. Wouldn't it be better to set `isCast` in `updateMyRate()` so it reflects whether I have *rated* rather than whether I have *clicked*? (In that case we should also rename the modifier to `m-show-results-cast`)